specs/experimental/op-contracts-manager.md

```diff
@@ -280,36 +271,45 @@ The `addGameType` method is used to orchestrate the actions required to add a ne
 or more chains.
 
 ```solidity
-struct PermissionlessGameConfig {
-  bytes32 absolutePrestate;
+struct AddGameInput {
+    string saltMixer;
+    ISystemConfig systemConfig;
+    IProxyAdmin proxyAdmin;
+    IDelayedWETH delayedWETH;
+    GameType disputeGameType;
+    Claim disputeAbsolutePrestate;
+    uint256 disputeMaxGameDepth;
+    uint256 disputeSplitDepth;
+    Duration disputeClockExtension;
+    Duration disputeMaxClockDuration;
+    uint256 initialBond;
+    IBigStepper vm;
+    bool permissioned;
+}
+
+struct AddGameOutput {
+    IDelayedWETH delayedWETH;
+    IFaultDisputeGame faultDisputeGame;
```

Comment:
I think it's fine to use the interface types here instead of `address`, but otherwise I think we should restrict ourselves to the elementary types
```suggestion
struct AddGameInput {
    string saltMixer;
    ISystemConfig systemConfig;
    IProxyAdmin proxyAdmin;
    IDelayedWETH delayedWETH;
    uint32 disputeGameType;
    bytes32 disputeAbsolutePrestate;
    uint256 disputeMaxGameDepth;
    uint256 disputeSplitDepth;
    uint64 disputeClockExtension;
    uint64 disputeMaxClockDuration;
    uint256 initialBond;
    IBigStepper vm;
    bool permissioned;
}

struct AddGameOutput {
    IDelayedWETH delayedWETH;
    IFaultDisputeGame faultDisputeGame;
```